src/components/BorrowerProfile/LendCta.vue

```diff
@@ -87,81 +87,61 @@
 								</kv-ui-select>
 
 								<!-- Sparkles wrapper -->
-								<div
-									class="tw-relative tw-inline-flex tw-flex-1"
-									:class="{'tw-w-full':isLendAmountButton}"
+								<complete-loan-wrapper
+									:is-complete-loan-active="isCompleteLoanActive"
+									:is-lend-amount-button="isLendAmountButton"
 								>
-									<!-- Lend button -->
-									<kv-ui-button
-										key="lendButton"
-										v-if="lendButtonVisibility && !isLessThan25"
-										class="tw-inline-flex tw-flex-1"
-										data-testid="bp-lend-cta-lend-button"
-										type="submit"
-									>
-										{{ ctaButtonText }}
-									</kv-ui-button>
+									<template #button>
 
-									<!-- Lend again/lent previously button -->
-									<kv-ui-button
-										key="lendAgainButton"
-										v-if="this.state === 'lent-to' && !isLessThan25"
-										class="tw-inline-flex tw-flex-1"
-										data-testid="bp-lend-cta-lend-again-button"
-										type="submit"
-										v-kv-track-event="[
-											'Lending',
-											'Add to basket',
-											'Lend again'
-										]"
-									>
-										Lend again
-									</kv-ui-button>
+										<!-- Lend button -->
+										<kv-ui-button
+											key="lendButton"
+											v-if="lendButtonVisibility && !isLessThan25"
+											class="tw-inline-flex tw-flex-1"
+											data-testid="bp-lend-cta-lend-button"
+											type="submit"
```

Comment:
Does this button need `v-kv-track-event`?

Reply:
+1